### PR TITLE
The Big Grunty PR

### PR DESCRIPTION
**Pull request by caelaislinn**
:cl: CaelAislinn
bugfix: Fixes some issues with grunt gear including better mask sprites and natural armour being detached, broken or missing
rscadd: Adds a respawning Unggoy Heavy job slot
tweak: Buffs Unggoy Ultra armour and shields but reduces slots to 1
/:cl:

### CODE REVIEW

File: code/modules/halo/clothing/marine.dm [hunk 1]
```diff
@@ -362,7 +362,7 @@
 	icon_base = null
 	item_flags = STOPPRESSUREDAMAGE|AIRTIGHT
 	can_hold_knife = 1
-	armor = list(melee = 45, bullet = 35, laser = 10, energy = 25, bomb = 30, bio = 100, rad = 50)
+	armor = list(melee = 40, bullet = 40, laser = 5, energy = 30, bomb = 15, bio = 0, rad = 0)
```
Comment (X0-11) [inline code/modules/halo/clothing/marine.dm:365]: What's this the armour for and why was it changed?

File: code/modules/halo/covenant/species/unggoy/armour.dm [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	species_restricted = list("Unggoy")
+	matter = list("nanolaminate" = 1)
+
+	//5 less all-round protection compared to marines and high ranking grunts
+	armor = list(melee = 50, bullet = 45, laser = 50, energy = 40, bomb = 35, bio = 25, rad = 25)
+
+/obj/item/clothing/suit/armor/special/unggoy_combat_harness/major
+	name = "Unggoy Combat Harness (Major)"
+	icon_state = "combatharness_major"
+	item_state = "combatharness_major"
```
Comment (X0-11) [inline code/modules/halo/covenant/species/unggoy/armour.dm:23]: now there's no difference between minors and majors :(

**Comment by mcalexev** [comment]
Small ideas from the Covvie discord. Could extra fuel rod guns/ammo be added to the unggoy heavy vendor for usage by the new heavies? Also add fuel rod ammo to the ammo fabricator! It's just repurposed industrial fuel cells after all, lore-wise.

**Comment by X0-11** [comment]
Ammo is specifically chosen and distributed cross faction, with the numbers mostly equalised. Ammunition fabrication would greatly disrupt that balance and quickly lead to spamming one single weapon.

**Comment by mcalexev** [comment]
Ahh, gotcha. Still, extra guns/ammo at the heavy vendor at least so the heavies can equip themselves. Perhaps a few plasma rifles too.

**Comment by X0-11** [comment]
There is a single heavy and a single fuel rod iirc. The same issue applies to adding more ammo and guns as with the fabricator

**Comment by mcalexev** [comment]
Righto! So it's just one heavy then. We thought it'd be multiple. Apologies for the confusion!

File: code/modules/halo/clothing/marine.dm (hunk 1 above)
Comment (caelaislinn) [reply code/modules/halo/clothing/marine.dm:365]: Marine EVA boots, I changed them to match the stats for standard marine boots

**Pull request merged**